controllers/datadogagent/utils_test.go

```diff
@@ -195,14 +195,15 @@ func Test_getVolumeMountsForSecurityAgent(t *testing.T) {
 			},
 		},
 		{
-			name: "compliance volumeMounts",
+			name: "runtime volumeMounts",
 			dda:  testutils.NewDatadogAgent("foo", "bar", "datadog/agent:7", &testutils.NewDatadogAgentOptions{SecuritySpec: securityRuntime}),
 			want: []v1.VolumeMount{
 				{Name: "logdatadog", ReadOnly: false, MountPath: "/var/log/datadog"},
 				{Name: "datadog-agent-auth", ReadOnly: true, MountPath: "/etc/datadog-agent/auth"},
 				{Name: "dsdsocket", ReadOnly: true, MountPath: "/var/run/datadog/statsd"},
 				{Name: "config", ReadOnly: false, MountPath: "/etc/datadog-agent"},
 				{Name: "hostroot", ReadOnly: true, MountPath: "/host/root"},
+				{Name: "runtimepoliciesdir", ReadOnly: true, MountPath: "/etc/datadog-agent/runtime-security.d"},
```

Comment:
/opt ?

Reply:
same here, it is on purpose that we mount in `/etc/` the emptyDir.